smoke-test/tests/tokens/revokable_access_token_test.py

```diff
@@ -0,0 +1,309 @@
+    assert res_data["data"]["listAccessTokens"]["tokens"][0]["tokenId"] == user_tokenId
+
+    # User should be able to revoke his own token
+    res_data = revokeAccessToken(user_session, user_tokenId)
+    assert res_data
+    assert res_data["data"]
+    assert res_data["data"]["revokeAccessToken"]
+    assert res_data["data"]["revokeAccessToken"] == True
+
+    # Using a normal account, check that all its tokens where removed.
+    res_data = listAccessTokens(user_session, [{"field": "actorUrn","value": "urn:li:corpuser:user"}])
+    assert res_data
+    assert res_data["data"]
+    assert res_data["data"]["listAccessTokens"]["total"] is not None
+    assert len(res_data["data"]["listAccessTokens"]["tokens"]) == 0
+
+    # Token created by user should be deletable by admin
+    res_data = generateAccessToken_v2(user_session, "urn:li:corpuser:user")
+    assert res_data
+    assert res_data["data"]
+    assert res_data["data"]["createAccessToken"]
+    assert res_data["data"]["createAccessToken"]["accessToken"]
+    assert res_data["data"]["createAccessToken"]["metadata"]["actorUrn"] == "urn:li:corpuser:user"
+    user_tokenId = res_data["data"]["createAccessToken"]["metadata"]["tokenId"]
+
+    # Admin can list
+    res_data = listAccessTokens(admin_session, [{"field": "actorUrn","value": "urn:li:corpuser:user"}])
+    assert res_data
+    assert res_data["data"]
+    assert res_data["data"]["listAccessTokens"]["total"] is not None
+    assert len(res_data["data"]["listAccessTokens"]["tokens"]) == 1
+    assert res_data["data"]["listAccessTokens"]["tokens"][0]["actorUrn"] == "urn:li:corpuser:user"
+    assert res_data["data"]["listAccessTokens"]["tokens"][0]["ownerUrn"] == "urn:li:corpuser:user"
+    assert res_data["data"]["listAccessTokens"]["tokens"][0]["tokenId"] == user_tokenId
+
+    # Admin can delete token created by someone else.
+    res_data = revokeAccessToken(admin_session, user_tokenId)
+    assert res_data
+    assert res_data["data"]
+    assert res_data["data"]["revokeAccessToken"]
+    assert res_data["data"]["revokeAccessToken"] == True
+
+    # Using a normal account, check that all its tokens where removed.
+    res_data = listAccessTokens(user_session, [{"field": "actorUrn","value": "urn:li:corpuser:user"}])
+    assert res_data
+    assert res_data["data"]
+    assert res_data["data"]["listAccessTokens"]["total"] is not None
+    assert len(res_data["data"]["listAccessTokens"]["tokens"]) == 0
+
+    # Using the super account, check that all tokens where removed.
+    res_data = listAccessTokens(admin_session, [{"field": "actorUrn","value": "urn:li:corpuser:user"}])
+    assert res_data
+    assert res_data["data"]
+    assert res_data["data"]["listAccessTokens"]["total"] is not None
+    assert len(res_data["data"]["listAccessTokens"]["tokens"]) == 0
+
+    # Normal user should not be able to generate token for another user
+    res_data = generateAccessToken_v2(user_session, "urn:li:corpuser:datahub")
```

Comment:
Minor comment: This is a big long test that is a bit confusing to follow. But good voerage which is nice!